docs/migration-guides/prometheus-migration.md

```diff
@@ -0,0 +1,328 @@
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CVX4CTZJ8VDVS2PQKQQYZVRN
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CW2XSE758NYFKWY2AX3ZGRMY
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CW8Q6AZPZKD10ZFFNP102V4Z
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CWEGJQ52W2YJXZ86J51FJGBP
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CWM9ZAP7SBZBXQJPT4409PE6
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CWT3C1RRY2SEZT93AX0EFYW0
+-rw-rw-r--    1 1001     1001             0 Nov 30 12:05 lock
+drwxrwsr-x    3 1001     1001          4096 Dec  5 12:01 wal
+```
+
+Now, kick the Prometheus pod with:
+
+`kubectl delete pod prometheus-0 -n kubeprod --grace-period=0`
+
+In the next start, Prometheus will restore the snapshot and you will see the following logs:
+
+```bash
+level=info ts=2018-11-30T13:02:09.537050487Z caller=main.go:238 msg="Starting Prometheus" version="(version=2.4.3, branch=HEAD, revision=167a4b4e73a8eca8df648d2d2043e21bdb9a7449)"
+level=info ts=2018-11-30T13:02:09.537122788Z caller=main.go:239 build_context="(go=go1.11.1, user=root@1e42b46043e9, date=20181004-08:42:02)"
+level=info ts=2018-11-30T13:02:09.537142375Z caller=main.go:240 host_details="(Linux 4.14.65+ #1 SMP Sun Sep 9 02:18:33 PDT 2018 x86_64 prometheus-0 (none))"
+level=info ts=2018-11-30T13:02:09.537159771Z caller=main.go:241 fd_limits="(soft=1048576, hard=1048576)"
+level=info ts=2018-11-30T13:02:09.537173945Z caller=main.go:242 vm_limits="(soft=unlimited, hard=unlimited)"
+level=info ts=2018-11-30T13:02:09.54195241Z caller=main.go:554 msg="Starting TSDB ..."
+level=info ts=2018-11-30T13:02:09.542280702Z caller=web.go:397 component=web msg="Start listening for connections" address=0.0.0.0:9090
+level=info ts=2018-11-30T13:02:09.557987841Z caller=repair.go:35 component=tsdb msg="found healthy block" mint=1540425600000 maxt=1540620000000 ulid=01CTTC16QP4NCPRM28H7Y6HXWB
+level=info ts=2018-11-30T13:02:09.566754786Z caller=repair.go:35 component=tsdb msg="found healthy block" mint=1540620000000 maxt=1540814400000 ulid=01CV05DT3E6QKSEMR2TAEEYWTQ
+level=info ts=2018-11-30T13:02:09.566884665Z caller=repair.go:35 component=tsdb msg="found healthy block" mint=1540814400000 maxt=1541008800000 ulid=01CV5YTF6Z4F22DJ5ECNM4PAH9
+level=info ts=2018-11-30T13:02:09.566960934Z caller=repair.go:35 component=tsdb msg="found healthy block" mint=1541008800000 maxt=1541203200000 ulid=01CVBR71YE4M97K3SDXXQ9449W
+level=info ts=2018-11-30T13:02:09.567034354Z caller=repair.go:35 component=tsdb msg="found 
+...
+level=info ts=2018-11-30T13:02:17.548873548Z caller=main.go:564 msg="TSDB started"
+```
+
+Now, the migration is done. Perform some queries to check the data loss.
+
+### Checking next TSDB compacts
+
+In order to ensure the snapshot was correctly done and the Prometheus TSDB is healthy and stable, you should check the Prometheus logs after several TSDB compacts.
```

Comment:
...the migration took place correctly and the new Prometheus TSDB is ...

Reply:
done, thank you.